subprojects/ce-login/celogin/include/CeLogin.h

```diff
@@ -240,26 +246,37 @@ struct AcfUserFields
  *  @param publicKeyLengthParm the length of the provided public key
  *  @param serialNumberParm a pointer to the serial number of the current system
  *  @param serialNumberLengthParm the length of the provided serial number
- *  @param expirationTimeParm a unix timestamp indicating when the ACF expires
+ *  @param currentReplayIdParm the current replay ID persisted by the BMC. The very first time
```

Comment:
You may want to be more specific, such as "for a brand new system"

First time could be interpreted as the first time this poweron